doc/man3/RAND_DRBG_reseed.pod

```diff
@@ -0,0 +1,84 @@
+=pod
+
+=head1 NAME
+
+RAND_DRBG_reseed,
+RAND_DRBG_set_reseed_interval,
+RAND_DRBG_set_reseed_time_interval
+- reseed the RAND_DRBG instance
+
+=head1 SYNOPSIS
+
+ #include <openssl/rand_drbg.h>
+
+ int RAND_DRBG_reseed(RAND_DRBG *drbg,
+                      const unsigned char *adin, size_t adinlen);
+
+ int RAND_DRBG_set_reseed_interval(RAND_DRBG *drbg,
+                                   unsigned int interval);
+
+ int RAND_DRBG_set_reseed_time_interval(RAND_DRBG *drbg,
+                                        time_t interval);
+
+
+=head1 DESCRIPTION
+
+These functions are part of the OpenSSL L<RAND_DRBG(7)> API.
+
+RAND_DRBG_reseed()
+reseeds the given B<drbg>,  mixing in the specified additional data provided
+in the buffer B<adin> of length B<adinlen>.
+The additional data can be omitted by setting B<adin>=NULL and B<adinlen>=0.
+
+RAND_DRBG_set_reseed_interval()
+sets the reseed interval of the B<drbg>, which is the maximum allowed number
+of generate requests between consecutive reseedings.
+If B<interval> > 0, then the B<drbg> will reseed automatically whenever the
+number of generate requests since its last seeding exceeds the given reseed
+interval.
+If B<interval> == 0, then this feature is disabled.
+
+
+RAND_DRBG_set_reseed_time_interval()
+sets the reseed time interval of the B<drbg>,  which is the maximum allowed
+number of seconds between consecutive reseedings.
+If B<interval> > 0, then the B<drbg> will reseed automatically whenever the
+elapsed time since its last reseeding exceeds the given reseed time interval.
+For safety, a reseeding will also occur if the clock has been reset to a
+smaller value.
```

Comment:
I think such implementation details shouldn't be mentioned.

Reply:
You mean the last sentence? I will remove it.